xml/installation-installation-smt-setup.xml

```diff
@@ -0,0 +1,238 @@
+     <guimenu>SCC E-Mail Used for Registration</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     <guimenu>Your SMT Server URL</guimenu> shows the HTTP address of your
+     server. Usually it should not be necessary to change it.
+    </para>
+   </step>
+   <step>
+    <para>
+     Select <guimenu>Next</guimenu> to proceed to step two of the <guimenu>SMT Configuration Wizard</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Enter a <guimenu>Database Password for SMT User</guimenu> and confirm
+     it by entering it once again.
+    </para>
+   </step>
+   <step>
+    <para>
+     Enter one or more e-mail addresses to which &smt; status reports are
+     sent by selecting <guimenu>Add</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Select <guimenu>Next</guimenu> to save your &smt; configuration. When
+     setting up the database you will be prompted for the MariaDB root
+     password. If you have not already created one then create it in this step. Note that this is
+     the global MariaDB root password, not the database password for the SMT
+     user you specified before.
+    </para>
+    <para>
+     The &smt; server requires a server certificate at
+     <filename>/etc/pki/trust/anchors/YaST-CA.pem</filename>. Choose
+     <guimenu>Run CA Management</guimenu>, provide a password and choose
+     <guimenu>Next</guimenu> to create such a certificate. If your
+     organization already provides a CA certificate, <guimenu>Skip</guimenu>
+     this step and import the certificate via <menuchoice>
+     <guimenu>YaST</guimenu> <guimenu>Security and Users</guimenu>
+     <guimenu>CA Management</guimenu> </menuchoice> after the &smt;
+     configuration is done. See
+   <link xlink:href="&suse-onlinedoc;/sles-12/book_security/data/cha_security_yast_ca.html"/>
+   for more information.
+    </para>
+    <para>
+     After you complete your configuration a synchronization check with the &scc; will run, which may take several minutes.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+
+ <sect1 xml:id="app.deploy.smt.repos">
+  <title>Setting up Repository Mirroring on the &smt; Server</title>
+
+  <para>
+   The final step in setting up the &smt; server is configuring it to
+   mirror the repositories needed for &cloud;. The &smt; server
+   mirrors the repositories from the &scc;. Make
+   sure to have the appropriate subscriptions registered in &scc; with the
+   same e-mail address you specified when configuring &smt;.
+  </para>
+
+  <sect2 xml:id="app.deploy.smt.repos.mandatory">
+   <title>Adding Mandatory Repositories</title>
+   <para>
+    Mirroring the &cloudos; and &productname; &productnumber;
+    repositories is mandatory. Run the following commands as user
+    &rootuser; to add them to the list of mirrored repositories:
+   </para>
+<screen><?dbsuse-fo font-size="0.63em"?>for REPO in &sle_repo;-{Pool,Updates} &cloud_repo;-{Pool,Updates}; do
+  smt-repos $REPO &smt_os; -e
```

Comment:
I think there is some command to enable by products (or all registered products). but this is fine as well

Reply:
It's not totally obvious to me what the product names are or how to discover that with the smt tools. I figured out by trial and error that SOC is SUSE-OpenStack-Cloud but for SLES I don't see how to go from the repo I want (e.g. SLES12-SP3-Pool) to the product name (SLES12-SP3? SLES12? SLES? SUSE-Linux-Enterprise?).